src/paste-markdown-html.ts

```diff
@@ -91,6 +95,17 @@ function convertToMarkdown(plaintext: string, walker: TreeWalker): string {
   return index === NODE_LIMIT ? plaintext : markdown
 }
 
+function isWithinUserMention(textarea: HTMLTextAreaElement): boolean {
+  const selectionStart = textarea.selectionStart || 0
```

Comment:
why not do
```
if (textarea.selectionStart === 0) {
  return false
}
```
at the beginning of the function, and then you wont need the if statements.

Reply:
Fixed!